gen/generate.go

```diff
@@ -188,7 +186,29 @@ func generateModule(m *compile.Module, i thriftPackageImporter, outDir string) e
 		files["types.go"] = buff
 	}
 
-	// TODO(abg): Services
+	if len(m.Services) > 0 {
+		for _, serviceName := range sortStringKeys(m.Services) {
+			service := m.Services[serviceName]
+			importPath, err := i.ServicePackage(service.ThriftFile(), service.Name)
+			if err != nil {
+				return err
+			}
+			packageName := filepath.Base(importPath)
+
+			// TODO inherited service functions
+
+			g := NewGenerator(i, importPath, packageName)
+			serviceFiles, err := Service(g, service)
+			if err != nil {
+				return fmt.Errorf("could not code for %s: %v", serviceName, err)
```

Comment:
I think you accidentally a word in "could not code for". also mention service maybe, "could not generate code code service %s"